docs/source/datasets.py

```diff
@@ -46,10 +46,67 @@
 }
 
 
+def sort_schemas(schemas: Tuple[str, DatasetSchema]) -> List[Union[str, DatasetSchema]]:
+    """Sort datasets (schemas) alphabetically.
+
+    Args:
+        schemas: A set containing a pair of schemas id (name) and DatasetSchema instance.
+
+    Returns:
+        A list of alphabetically ordered schemas instances based on their schema id (name).
+    """
+    return sorted(schemas, key=lambda schema: schema[0])
```

Comment:
When `sorted` can be applied to the incoming argument `schemas`, that suggests it is a list (or some other kind of iterable).

Reply:
I think input and output type should be `List[Tuple[str, DataseSchema]]`.

Reply:
You are correct, good one